### ISSUE TITLE
Some simple improvements to the calculator

### ISSUE THREAD

**Issue by rdstern** [africanmathsinitiative/R-Instat#7800]
@anastasia-mbithe I hope you could do this.  I am writing the possible video script called **_Fun with numbers_** that uses this dialogue.

a) In our new integer keyboard, the binary key includes an as.integer function at the start.  This has the problem that all integers more than just over 1000 come as NA.  Try with a data frame of 10,000 (say) and the first variable being 1, 2, 3, ... (That's now easy to produce with the File > New Data Frame).
I've been looking to define it differently, because if you just omit the `as.integer`, then it comes as a character variable.  But I can't find the appropriate type, so I think we'll have to put up with it being a character.  Happy if you could explore too.
b) In the main keyboard could you add spaces round +, -, * and /.  So pressing them gives <space>+<space>, etc.
c) In the probability keyboard there is pbirth and qbirth.  pbirth currently gives `pbirthday( )`.  But that doesn't accept a column, just a number.  So please change it to `sapply( ,pbirthday)` with the cursor in the space before the comma.  That will allow a variable to be given.  Same for qbirth.  Check with a column of integer values.  (And with fractions for qbirth.)

**Comment by rdstern** [comment]
I have also found a bug or two in the calculator - in the development version - it is ok in Version 0.7.7.  
![image](https://user-images.githubusercontent.com/11226469/185849330-7eae2a5c-c95e-411c-97de-23788f2c8a7f.png)

The error in the formula is simply that the bracket is omitted.  The error used to be given in the popup.

Now very puzzling but the bug then appeared in version 7.7 and now it all seems to be ok.  So I do wonder if it is interacting with the language issues in some way?

I assume this is nothing to do with a change that has recently been added to the French - but mention this here also, just in case.  I was using the version initially in French and now moved back to English.  @lloyddewit the comment at the bottom remains in French.  I have tried going into Portuguese and it remains very happy to stay in French.

And another bug that seems more straightforward to solve:
![image](https://user-images.githubusercontent.com/11226469/186114085-9cc09530-a35c-4223-ae98-4cde6e3a7834.png)

I was using the calculator in the default way, i.e. where it is now set to write a column to the grid.  Then I turned that off, because I just wanted a single line of output in the results window.  But it wrote a column - called last_calc.  When I used it again - it didn't - as you can see from the code and output copied above.  It generated much simpler code the second time.

**Comment by lloyddewit** [comment]
>  I was using the version initially in French and now moved back to English. @lloyddewit the comment at the bottom remains in French. I have tried going into Portuguese and it remains very happy to stay in French.

@rdstern I'm a bit puzzled by this. The comment translation is implemented in PR #7783 but this PR is not yet merged (waiting for @N-thony 's review/approval). If you are testing on the latest master, then the comment should always be in English.
Are you testing on a different branch? Or did you change the comment manually?
thanks

**Comment by rdstern** [comment]
I have been using various pull requests and the master.  Now I am in English and using Patrick's recent changes to the output window.  I haven't ever edited the comment - at least not recently.  Here is a dialogue:
![image](https://user-images.githubusercontent.com/11226469/186647926-7a95ff9e-50fe-4473-9a3f-3af1855be2a3.png)

**Comment by lloyddewit** [comment]
@rdstern 
The comment translation is not implemented on the branches you tested. I assume that the last time you tested PR #7783, you exited in French. R-Instat then stored the French comment as the default. When you restarted R-Instat in the master branch, then R-Instat reloads the last specified comment (in this case the French comment). However, master branch doesn't have the comment translation implemented yet so it can't convert the French comment to English. If you test in the PR #7783 branch, then everything should work correctly.
In summary, this issue should be resolved when PR #7783 is merged.

I hope that makes sense, if not, then let me know.

Thanks!

**Comment by rdstern** [comment]
@lloyddewit great - thanks.
